src/web/app/pages-help/instructor-help-page/instructor-help-courses-section/instructor-help-courses-section.component.scss

```diff
@@ -5,3 +5,9 @@
 .instr-help-qn-last {
   padding-bottom: 5px;
 }
+
+@media screen {
+  button {
+    text-align: left;
+  }
```

Comment:
What is the reason to have @media here?

Reply:
@media has been removed, initially I thought it should depend on whether it was on screen or not.